packages/dynamoose/lib/Model/index.ts

```diff
@@ -407,6 +406,12 @@ export class Model<T extends ItemCarrier = AnyItem> extends InternalPropertiesCl
 		return this.getInternalProperties(internalProperties).name;
 	}
 
+	get tableName (): string {
```

Comment:
Slightly confused about the purpose behind this.

Reply:
Model.tableName is now a getter instead of an optional string parameter of ModelInternalProperties.
The code of this getter allows to always get the correct name of the model table, even if the table itself has not been initialised yet